ui/src/app/applications/components/application-status-panel/application-status-panel.tsx

```diff
@@ -272,6 +272,16 @@ export const ApplicationStatusPanel = ({application, showDiff, showOperation, sh
                         <div className='application-status-panel__item-name' style={{marginBottom: '0.5em'}}>
                             {appOperationState.phase} <Timestamp date={appOperationState.finishedAt || appOperationState.startedAt} />
                         </div>
+                        <div className='application-status-panel__item-name' style={{marginBottom: '0.5em'}}>
```

Comment:
The LastRefresh is what will cause an application to be OutOfSync. It should be related to the sync status, not to the Last Sync.